src/AtmosphereModels/atmosphere_model.jl

```diff
@@ -112,7 +114,7 @@ function AtmosphereModel(grid;
 
     density = materialize_density(formulation, grid)
     velocities, momentum = materialize_momentum_and_velocities(formulation, grid, boundary_conditions)
-    tracers = NamedTuple(n => CenterField(grid, boundary_conditions=boundary_conditions[n]) for name in tracers)
+    tracers = NamedTuple(n => CenterField(grid, boundary_conditions=boundary_conditions[n]) for n in tracers)
```

Comment:
Let me guess, this wasn't covered by tests before? Is it now?

Reply:
i can add